UnityProject/Assets/Scripts/UI/Systems/AdminTabButtons.cs

```diff
@@ -79,12 +79,12 @@ public void BtnOpenTileVV()
 
 		private void DisableAllGUI()
 		{
-			adminTools.gameObject.SetActive(false);
-			devSpawner.gameObject.SetActive(false);
-			devCloner.gameObject.SetActive(false);
-			devDestroyer.gameObject.SetActive(false);
-			devSelectTile.gameObject.SetActive(false);
-			tileChanger.Close();
+			//adminTools.gameObject.SetActive(false);
```

Comment:
why are these commented?

Reply:
is, so you can have multiple mapping tools open at the same time

Reply:
then your intention is to change the behaviour and you should change the code to represent that. Just commenting out this leaves a functionality you don't want anymore lingering in the codebase without explanation

Reply:
ok